Python/pywarpx/picmi.py

```diff
@@ -734,6 +735,7 @@ def init(self, kw):
         self.costs_heuristic_cells_wt = kw.pop('warpx_costs_heuristic_cells_wt', None)
         self.use_fdtd_nci_corr = kw.pop('warpx_use_fdtd_nci_corr', None)
         self.amr_check_input = kw.pop('warpx_amr_check_input', None)
+        self.amr_checkpoint = kw.pop('warpx_amr_checkpoint', None)
```

Comment:
I would use consistent naming and have the input be `warpx_amr_restart` since it will be assigned to `amr.restart`.